src/Components/Search/__tests__/SearchBar.test.tsx

```diff
@@ -23,6 +23,12 @@ const searchResults = {
       },
     ],
   },
+
+  filter_artworks: {
```

Comment:
cc @pepopowitz we were wondering the other day if we could write a test here that highlights a suggestion, and asserts the preview is rendered.

Looks like we should have been able to! And so we must not have been triggering the suggestion highlighting properly.

This PR adds in the detail from the comp that renders the merch artworks preview and 'first item placeholder' as you trigger suggestions, and so stubbing it in this way seems to work!